.github/workflows/e2e_nightly.yml

```diff
@@ -34,6 +34,28 @@ jobs:
       contents: read
       packages: write
 
+  gpu-test-matrix:
+    strategy:
+      matrix:
+        platform:
+          - name: K3s-QEMU-SNP-GPU
+            runner: SNP
+            self-hosted: true
+        test-name: [servicemesh, openssl, policy, workloadsecret, volumestatefulset, gpu]
```

Comment:
Do we really need to run all those tests for GPU?

Reply:
I would like to also run tests that exercise service mesh functionality, etc. on the GPU runtime class.